components/ingest-service/migration/a1_migration.go

```diff
@@ -100,6 +103,18 @@ func (ms *Status) stage2() {
 	ms.finalCleanup()
 	ms.taskCompleted()
 
+	err := ms.SendAllActionsThroughPipeline()
+	if err != nil {
+		logFatal(err.Error(), "Unable to re-insert actions")
+	}
+	ms.taskCompleted()
+
+	err = ms.DeleteAllActionsIndexes()
+	if err != nil {
+		logFatal(err.Error(), "Unable to delete action indexes")
+	}
+	ms.taskCompleted()
+
```

Comment:
After all the A1 actions are updated to A2, we re-ingest the actions to send them to the event-feed-service and policy information to config-mgmt-service.